torch_geometric/graphgym/contrib/layer/generalconv.py

```diff
@@ -54,7 +53,7 @@ def norm(edge_index, num_nodes, edge_weight=None, improved=False,
             edge_index, edge_weight, fill_value, num_nodes)
 
         row, col = edge_index
-        deg = scatter_add(edge_weight, row, dim=0, dim_size=num_nodes)
+        deg = scatter(edge_weight, row, dim=0, dim_size=num_nodes)
```

Comment:
Should we leave this as `scatter_sum` or add it as the `reduce` argument? I feel its quite nice for us to be explicit about the reduce method?

Reply:
In general, I think it's better to have a single function in API (`scatter`) - more options raise confusion. I didn't have a strong preference for setting or not the `reduce='sum'` (default, most common option), but now after giving it a 2nd thought, I agree with you - we should set the reduction algorithm explicitly throughout the code.

Reply:
I agree having a single function is better 👍